app/models/ingredient.rb

```diff
@@ -1,3 +1,10 @@
 class Ingredient < ApplicationRecord
-   validates_presence_of :name, :cost
+  has_many :recipe_ingredients
+  has_many :recipes, through: :recipe_ingredients
+
+  validates_presence_of :name, :cost
+
+  def self.ingredients_sorted
+    all.order(:name)
```

Comment:
I think you can either call `self.order` or just `order`. This is a class level method so it will perform the operation on all of them.